.github/workflows/test-backends.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         id: set-matrix
         run: |
           if [ x"${{ github.repository }}" == x"openai/triton" ]; then
-            echo '::set-output name=matrix-optional::[["self-hosted", "gfx908"], ["self-hosted", "arc770"]]'
+            echo '::set-output name=matrix-optional::[["self-hosted", "rocm"], ["self-hosted", "arc770"]]'
```

Comment:
is "rocm" the right tag here? how would we distinguish MI210 from other GPU types in the future?

Reply:
That's a good question. Let me confirm with Omkar on the labels.

Reply:
Can we use "rocm" for now. And later when we add MI300 Omkar will change labels to include arch.